txtorcon/torinfo.py

```diff
@@ -66,6 +66,14 @@ def dump(self, prefix):
             x.dump(prefix)
 
 
+def _get_descriptor(data):
+    try:
+        from stem.descriptor.server_descriptor import RelayDescriptor
+        data = RelayDescriptor(data)
+    except ImportError:
+        pass
+    return data
+
 class ConfigMethod(object):
```

Comment:
This all looks pretty good (haven't actually tried it yet, though)!

I think probably the ImportError should just be allowed to happen, as you only add ``_get_descriptor`` if .use_stem was True -- and then it's an error if you don't have Stem installed, I'd say? Also maybe "_parse_stem_descriptor" or something instead for a helper name, since we're not really getting it here (just parsing).

Thanks!